### PR TITLE
std_mean.py for benchmark stats

### PR DESCRIPTION
**Pull request by will-moore**
This prints stats like:

```
2d_benchmark_data.csv

Mean
                 duration  chunk_distance  round   seconds
type     source                                           
HDF5     http    0.221371       266181.95   49.5  0.221113
         local   0.002998       266181.95   49.5  0.002818
         s3      1.122068       266181.95   49.5  1.121805
Overhead http    0.001425       266181.95   49.5  0.001269
         local   0.000134       266181.95   49.5  0.000014
         s3      0.011508       266181.95   49.5  0.011279
TIFF     http    0.151340       266181.95   49.5  0.151114
         local   0.086487       266181.95   49.5  0.086267
         s3      0.388529       266181.95   49.5  0.388272
Zarr     http    0.006926       266181.95   49.5  0.006652
         local   0.007388       266181.95   49.5  0.007099
         s3      0.131854       266181.95   49.5  0.131575

Std
                 duration  chunk_distance      round   seconds
type     source                                               
HDF5     http    0.051123   149981.733567  29.011492  0.051115
         local   0.004189   149981.733567  29.011492  0.004156
         s3      0.322672   149981.733567  29.011492  0.322666
Overhead http    0.001197   149981.733567  29.011492  0.001187
         local   0.000016   149981.733567  29.011492  0.000002
         s3      0.002839   149981.733567  29.011492  0.002838
TIFF     http    0.037332   149981.733567  29.011492  0.037327
         local   0.036226   149981.733567  29.011492  0.036227
         s3      0.088530   149981.733567  29.011492  0.088532
Zarr     http    0.001773   149981.733567  29.011492  0.001760
         local   0.002866   149981.733567  29.011492  0.002868
         s3      0.019592   149981.733567  29.011492  0.019609

3d_benchmark_data.csv

Mean
                 duration  chunk_distance  round   seconds
type     source                                           
HDF5     http    0.220858      1707871.23   49.5  0.220592
         local   0.002702      1707871.23   49.5  0.002479
         s3      1.046434      1707871.23   49.5  1.046130
Overhead http    0.001301      1707871.23   49.5  0.001163
         local   0.000150      1707871.23   49.5  0.000023
         s3      0.012821      1707871.23   49.5  0.012607
TIFF     s3           NaN       379642.00   99.0  0.928801
Zarr     http    0.013514      1707871.23   49.5  0.013290
         local   0.007850      1707871.23   49.5  0.007667
         s3      0.100792      1707871.23   49.5  0.100552
Std
                 duration  chunk_distance      round   seconds
type     source                                               
HDF5     http    0.051433    906299.64163  29.011492  0.051430
         local   0.002911    906299.64163  29.011492  0.002880
         s3      0.259094    906299.64163  29.011492  0.259042
Overhead http    0.000518    906299.64163  29.011492  0.000509
         local   0.000086    906299.64163  29.011492  0.000051
         s3      0.005196    906299.64163  29.011492  0.005179
TIFF     s3           NaN             NaN        NaN       NaN
Zarr     http    0.009346    906299.64163  29.011492  0.009336
         local   0.006391    906299.64163  29.011492  0.006381
         s3      0.015169    906299.64163  29.011492  0.015172
```

cc @jburel @joshmoore

### CODE REVIEW

File: results/std_mean.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Get the standard deviation and mean for all the benchmark data
+# grouped by type (e.g. HDF5, TIFF, Zarr, Overhead) and by
+# source (e.g. http, local, s3)
+
+for csv_file in ["2d_benchmark_data.csv", "3d_benchmark_data.csv"]:
+
+    print(csv_file)
+
+    df = pandas.read_csv("2d_benchmark_data.csv")
```
Comment (joshmoore) [inline results/std_mean.py:11]:
```suggestion
    df = pandas.read_csv(csv_file)
```

Comment (joshmoore) [reply results/std_mean.py:11]: Otherwise, :+1:

**Comment by will-moore** [comment]
Oops - thanks for catching my typo @joshmoore.
Updated the output in the description.

**Comment by joshmoore** [comment]
A heads up that I'm having a hard time getting the build to pass. (2 retries so far)

**Comment by joshmoore** [comment]
Can't really be related so merging. A future nice-to-have might be would be to order as in the figure.

**Pull request merged**